src/main/resources/com/cloudbees/jenkins/plugins/amazonecs/ECSTaskTemplate/config.jelly

```diff
@@ -31,6 +31,9 @@
   <f:entry title="${%Template Name}" field="templateName" description="The name that will be appended to the ECS cluster name when creating task definitions. Cannot be used with a Task Definition Override.">
       <f:textbox />
     </f:entry>
+  <f:entry title="${%Agent Container Name}" field="agentContainerName" description="Name of the jenkins agent container. This is value is only used when Task Definition Override is defined.">
```

Comment:
We might consider capitalizing "Jenkins" here—I'd expect that to be requested upstream.

Reply:
Done.